addon/templates/components/basic-dropdown-trigger.hbs

```diff
@@ -4,21 +4,13 @@
     {{if @renderInPlace " ember-basic-dropdown-trigger--in-place"}}
     {{if @hPosition (concat " ember-basic-dropdown-trigger--" @hPosition)}}
     {{if @vPosition (concat " ember-basic-dropdown-trigger--" @vPosition)}}
-
     {{@defaultClass}}"
```

Comment:
Could use some feedback on this one. I've opted not to include the above class names in the modifier. I felt that if a user is implementing a custom trigger using the modifier, they have direct access to the element so they can apply any class names they want to apply

Let me know if I should include them. I could do that just to be safe.